portfolio/src/main/webapp/index.html

```diff
@@ -52,11 +52,25 @@ <h1>Michaela Murray's Portfolio</h1>
 	       <span class="mdc-button__ripple">Resume</span> 
 	    </button>
           </form>
-        </div> 
-        <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+        </div>
+	<div id="icons" style="display:block">
+            <ul style="list-style-type:none;margin-left:-42px;"> 
+            <li style="display:inline-block;"> 
```

Comment:
You're adding styles directly to your elements but in other cases you're using css. i'm still not sure what is the right way. Consider checking appropriate guides.

Reply:
Hi Kaveh, I checked the style guide, and according to this point here: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, the style should be completely confined to the css file! I'll address that in my next commit-thanks!